README.md

```diff
@@ -49,19 +49,19 @@ $ bundle exec rake
 ## Adding a new specialist document format
 
 1. Add the document_type to the `document_types` array in `config/routes.rb`
-2. Add a controller that inherits `AbstractDocumentController`
+2. Add a controller that inherits `AbstractDocumentsController`
 3. Add the schema to the `finders/schemas` folder and define the singleton for it in `app/lib/specialist_publisher_wiring.rb`
 4. Add the metadata about the Finder to `finders/metadata`
 5. Add a model (which is a subclass of `DocumentMetadataDecorator` and only defines the extra fields of the document type), validator and builder for the new format.
 6. Define the factory with the builder in `app/lib/specialist_publisher_wiring.rb`.
 7. Define the validatable document factory in `app/models/document_factory_registry.rb`
 8. Define a repository in `app/repositories/repository_registry.rb`
-9. Add observers, along with formatters required:
-  - `document_type_publication_alert_formatter.rb` in `app/exporters/formatters/`
-  - `document_type_artefact_formatter.rb` in `app/lib/` for Panopticon
-  - `document_type_indexable_formatter.rb` app/lib/` for Rummager
-  - define a factory for `document_type_panopticon_registerer`, `document_type_rummager_indexer`, `document_type_rummager_deleter` and `document_type_content_api_exporter` in `app/lib/specialist_publisher_wiring.rb`
-  - add an Observers registry for the document type and add it to the hash in the `observers_registry` method in `app/lib/specialist_publisher`
+9. Add observers, along with formatters required. In in `app/exporters/formatters/`:
+  - `document_type_publication_alert_formatter.rb` 
```

Comment:
I know it's not something you've changed, but (assuming I'm correct - @tommyp?) could you append `for email-alert-api`?